visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -786,34 +912,128 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
+                            </div>
+                            <div className={classNames.tooltipTable}>
+                                <div className={classNames.tooltipColumn}>
+                                    <Text className={classNames.boldText}>
+                                        {localization.WhatIfTab.classPickerLabel}
+                                    </Text>
+                                    {tooltipClasses}
+                                </div>
+                                <div className={classNames.tooltipColumn}>
+                                    <Text block className={classNames.boldText}>
+                                        {localization.WhatIfTab.probabilityLabel}
+                                    </Text>
+                                    {tooltipProbs}
+                                </div>
+                            </div>
+                        </div>
+                    ),
+                };
+                return (
+                    <div className={classNames.predictedBlock}>
+                        <TooltipHost
+                            tooltipProps={tooltipProps}
+                            delay={TooltipDelay.zero}
+                            id={WhatIfTab.basePredictionTooltipIds}
+                            directionalHint={DirectionalHint.leftCenter}
+                            styles={{ root: { display: 'inline-block' } }}
+                        >
+                            <IconButton
+                                className={classNames.tooltipHost}
+                                iconProps={{ iconName: 'More' }}
+                            ></IconButton>
+                        </TooltipHost>
+                        <div>
+                            {trueClass !== undefined && (
+                                <div>
+                                    <Text className={classNames.boldText} variant="small">
+                                        {localization.WhatIfTab.trueClass}
+                                    </Text>
+                                    <Text variant="small">
+                                        {
+                                            this.props.jointDataset.metaDict[JointDataset.PredictedYLabel]
+                                                .sortedCategoricalValues[trueClass]
+                                        }
+                                    </Text>
+                                </div>
                             )}
-                        </Text>
-                    )}
+                            <div>
+                                <Text className={classNames.boldText} variant="small">
+                                    {localization.WhatIfTab.predictedClass}
+                                </Text>
+                                <Text variant="small">{predictedClassName}</Text>
+                            </div>
+                            <div>
```

Comment:
should we be using Stack instead of divs here?

Reply:
I've had some bad experiences with Stack support in Edge/Safari, and don't find the utility of stack to be advantageous to just flex.